docs/.vitepress/showcases.ts

```diff
@@ -222,7 +222,7 @@ export const showcases: ShowCaseInfo[] = [
     },
     at: 'FOSSASIA Summit 2024',
     slidesLink: 'https://talk.quan.hoabinh.vn/rust-error-handling-ease-web-dev/',
-    sourceLink: 'https://hongquan@bitbucket.org/hongquan/rust-error-handling-ease-web-dev.git',
```

Comment:
Seems like it should be `https://bitbucket.org/hongquan/rust-error-handling-ease-web-dev/src/main/`

I merged it too fast 🫠